redshift.tf

```diff
@@ -6,5 +6,12 @@ resource "aws_redshift_cluster" "redshiftEncryptedFalse" {
   node_type          = "dc1.large"
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure Redshift Cluster Audit Logging is Enabled**
 
**Vulnerability**
Ensure audit logging is enabled for Redshift clusters for security and troubleshooting purposes.
 
**Remediation**
Once enabled, the Amazon Redshift Audit Logging feature starts recoding database usage information such as queries performed and connection attempts, logging data that can be extremely useful for security and compliance audits or troubleshooting sessions.